modules/rostests/apitests/user32/GetDCEx.c

```diff
@@ -247,15 +244,119 @@ Test_GetDCEx_CS_CLASSDC()
 
     DestroyWindow(hwnd1);
     DestroyWindow(hwnd2);
-    ok(UnregisterClass(pszClassName, GetModuleHandleA(0)) == TRUE,
+    ok(UnregisterClassW(pszClassName, GetModuleHandleA(0)) == TRUE,
        "UnregisterClass failed");
 }
 
 static
 void
-Test_GetDCEx_CS_Mixed()
+Test_GetDCEx_CS_CLASSDC_NEXT(void)
+{
+    static const LPCWSTR pszClassName = L"TestClass_CS_SwitchedStyle";
+    HWND hwnd1, hwnd2;
+    HDC hdc1, hdc2 , hdcClass;
+
+    //1
```

Comment:
An explanation of what's being tested would be welcome (for each of the "1", ..., "4" tests)

Reply:
As I write below, I now have the problem that these tests are crashing Windows XP, so far I've fixed it provisionally, but it's a bad solution, if no one thinks of anything I'll remove these tests. Anyway, I'm on various cases on trying to figure out exactly how this works in Windows. If the tests can be fixed to work I'll try to describe them, they are combinations on which it should be noticeable how hwnd changes in DC with different order of operations.
Anyway, so far I haven't found any differences between Windows XP and 11 in the GetDC results.